src/pages/About/AboutKyberSwap/index.tsx

```diff
@@ -61,7 +61,7 @@ import { useDarkModeManager } from 'state/user/hooks'
 import { ExternalLink, StyledInternalLink } from 'theme'
 import { formatBigLiquidity } from 'utils/formatBalance'
 
-import KyberSwapGeneralIntro from './KyberSwapGeneralIntro'
+import KyberSwapGeneralIntro from '../KyberSwapGeneralIntro'
```

Comment:
Better to not using `../`. Use absolute link import instead.

Reply:
Not quite needed, as I'm importing something from the same package/context.

Using absolute is better when I need to import something from some common packages or there are too many `..`.

In this case, `..` is absolutely fine. `KyberSwapGeneralIntro` and this `index.tsx` files will be in the same package together forever, so... please ignore this one.